src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -680,13 +666,6 @@ private File overwriteOrCreateWorkspace() {
         }
     }
 
-    private static void checkIfValidWorkspace(final File workspaceDir) {
-        final File tempFile = new File(workspaceDir.getAbsolutePath() + "/__tiledb_workspace.tdb");
```

Comment:
Is this check performed internally now?

Reply:
Yes, this check is performed internally in GenomicsDB now.